UnityProject/Assets/Scripts/UI/Jukebox/GUI_Jukebox.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Objects;
+using UnityEngine;
+
+public class GUI_Jukebox : NetTab
+{
+	[SerializeField]
+	private NetLabel labelSong = null;
+
+	[SerializeField]
+	private NetLabel labelArtist = null;
+
+	[SerializeField]
+	private NetLabel labelTrack = null;
+
+	[SerializeField]
+	private NetPrefabImage prefabImagePlayStop = null;
+
+	[SerializeField]
+	private NetSlider sliderVolume = null;
+
+	private Jukebox _JukeboxController;
+	private Jukebox jukeboxController
+	{
+		get
+		{
+			if (_JukeboxController == null)
+				_JukeboxController = Provider.GetComponent<Jukebox>();
+
+			return _JukeboxController;
+		}
+	}
+
+	public async void OnTabOpenedHandler(ConnectedPlayer connectedPlayer)
```

Comment:
Codacy found an issue: [Return 'Task' instead.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)

Reply:
That class should not exist anymore it's moved into UI/Object.  Why is it still there?
Deleted on my branch.
Fixed, will commit soon.